packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -303,7 +304,7 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
                     ref,
                     role: "combobox",
                 },
-                this.props.customTarget != null ? this.props.customTarget : this.getTagInput(listProps),
+                this.props.renderTarget != null ? this.props.renderTarget(targetProps) : this.getTagInput(listProps),
```

Comment:
I don't think we want to pass `targetProps` since we're already wrapping the custom target in an element that is handling spreading those.

I think the idea is to have a `renderTarget` callback that is passed `selectedItems` to ensure we always are rendering an up to date target. We could also use that callback to pass anything else that could be helpful, such as passing `isOpen` if the user would want to style differently based on that without separately keeping track of the loading state.